src/parsers/dr_ste_v1_tag_parser.py

```diff
@@ -0,0 +1,450 @@
+	ret["second_vlan_qualifier"] = _val(bin_str[110: 112])
+	ret["second_priority"] = _val(bin_str[112: 115])
+	ret["second_cfi"] = _val(bin_str[115: 116])
+	ret["second_vlan_id"] = _val(bin_str[116: 128])
+	return ret
+
+
+def mlx5_ifc_ste_eth_l2_dst_v1_bits_tag_parser_p(bin_str) :
+	ret = {}
+	ret["reserved_at_0"] = _val(bin_str[0: 1])
+	ret["sx_sniffer"] = _val(bin_str[1: 2])
+	ret["functional_lb"] = _val(bin_str[2: 3])
+	ret["ip_fragmented"] = _val(bin_str[3: 4])
+	ret["qp_type"] = _val(bin_str[4: 6])
+	ret["encapsulation_type"] = _val(bin_str[6: 8])
+	ret["port"] = _val(bin_str[8: 10])
+	ret["l3_type"] = _val(bin_str[10: 12])
+	ret["l4_type"] = _val(bin_str[12: 14])
+	ret["first_vlan_qualifier"] = _val(bin_str[14: 16])
+	ret["first_priority"] = _val(bin_str[16: 19])
+	ret["first_cfi"] = _val(bin_str[19: 20])
+	ret["first_vlan_id"] = _val(bin_str[20: 32])
+	ret["dmac"] = _val(bin_str[32: 80])
+	ret["l3_ethertype"] = _val(bin_str[80: 96])
+	ret["reserved_at_60"] = _val(bin_str[96: 102])
+	ret["tcp_syn"] = _val(bin_str[102: 103])
+	ret["reserved_at_67"] = _val(bin_str[103: 106])
+	ret["force_lb"] = _val(bin_str[106: 107])
+	ret["l2_ok"] = _val(bin_str[107: 108])
+	ret["l3_ok"] = _val(bin_str[108: 109])
+	ret["l4_ok"] = _val(bin_str[109: 110])
+	ret["second_vlan_qualifier"] = _val(bin_str[110: 112])
+	ret["second_priority"] = _val(bin_str[112: 115])
+	ret["second_cfi"] = _val(bin_str[115: 116])
+	ret["second_vlan_id"] = _val(bin_str[116: 128])
+	return ret
+
+
+def mlx5_ifc_ste_eth_l2_src_dst_v1_bits_tag_parser(bin_str) :
+	ret = {}
+	ret["dmac_47_16"] = _val(bin_str[0: 32])
+	ret["smac_47_16"] = _val(bin_str[32: 64])
+	ret["dmac_15_0"] = _val(bin_str[64: 80])
+	ret["reserved_at_50"] = _val(bin_str[80: 82])
+	ret["functional_lb"] = _val(bin_str[82: 83])
+	ret["reserved_at_53"] = _val(bin_str[83: 88])
+	ret["port"] = _val(bin_str[88: 90])
+	ret["l3_type"] = _val(bin_str[90: 92])
+	ret["reserved_at_5c"] = _val(bin_str[92: 94])
+	ret["first_vlan_qualifier"] = _val(bin_str[94: 96])
+	ret["first_priority"] = _val(bin_str[96: 99])
+	ret["first_cfi"] = _val(bin_str[99: 100])
+	ret["first_vlan_id"] = _val(bin_str[100: 112])
+	ret["smac_15_0"] = _val(bin_str[112: 128])
+	return ret
+
+
+def mlx5_ifc_ste_eth_l2_src_dst_v1_bits_tag_parser_p(bin_str) :
+	ret = {}
+	ret["dmac_47_16"] = _val(bin_str[0: 32] + bin_str[64: 80])
+	ret["smac_47_16"] = _val(bin_str[32: 64] + bin_str[112: 128])
+	ret["reserved_at_50"] = _val(bin_str[80: 82])
+	ret["functional_lb"] = _val(bin_str[82: 83])
+	ret["reserved_at_53"] = _val(bin_str[83: 88])
+	ret["port"] = _val(bin_str[88: 90])
+	ret["l3_type"] = _val(bin_str[90: 92])
+	ret["reserved_at_5c"] = _val(bin_str[92: 94])
+	ret["first_vlan_qualifier"] = _val(bin_str[94: 96])
+	ret["first_priority"] = _val(bin_str[96: 99])
+	ret["first_cfi"] = _val(bin_str[99: 100])
+	ret["first_vlan_id"] = _val(bin_str[100: 112])
+	return ret
+
+
+def mlx5_ifc_ste_eth_l3_ipv4_5_tuple_v1_bits_tag_parser(bin_str) :
```

Comment:
not used

Reply:
the ..._p (pretty) is used, usually we keep both structs

Reply:
what's the difference?